### ISSUE TITLE
Convert weights for use with Theano backend and TH dim ordering

### ISSUE THREAD

**Issue by kentsommer** [kentsommer/keras-inceptionV4#2]
Since Tensorflow and Theano implement convolutions in different ways it is necessary to port a new set of weights for use with the Theano backend with TH dim ordering.

**Comment by kentsommer** [comment]
Per the changes in c5e47d1dcd1f96089798f581e20fa7911a0e1681, the convolution weights are fully converted for theano with th dim ordering.

However, according to @fchollet it is also necessary to row-shuffle the weights of the Dense layer. 

I have tried a few things without success already, but will continue to work on this as I have time or until someone else finds a solution (PR's welcome!).

**Comment by kentsommer** [comment]
This issue has been solved, see 90d71706bc708602b40db0e9ee9c52220f4ec2ca for details.
